srcs/runners_manager/vm_creation/CloudManager.py

```diff
@@ -70,6 +70,10 @@ def delete_vm(self, runner: Runner):
     def delete_images_from_shelved(self, name):
         raise NotImplementedError
 
+    @abc.abstractmethod
+    def add_labels_to_instance(self, instance_name: str, labels_webhook: dict):
```

Comment:
as you got confused with tags and labels because each cloud has its own way of naming things, I suggest you to review the name of this function with this in mind.

Reply:
Done ! I used the word `identifier`